kernel/kernel-impl/src/main/java/org/sakaiproject/site/impl/BaseSiteService.java

```diff
@@ -3497,6 +3498,11 @@ else if (EVENT_SITE_VISIT_DENIED.equals(eventType) || AuthzGroupService.SECURE_U
 		{
 			clearUserCacheForUser(event.getUserId());
 		}
+		else if(SiteService.SECURE_UPDATE_SITE_MEMBERSHIP.equals(eventType) || SiteService.SECURE_UPDATE_GROUP_MEMBERSHIP.equals(eventType)
+				|| SiteService.SECURE_UPDATE_SITE.equals(eventType))
+		{
+            SiteParticipantNotification.notifySiteParticipant("/gradebook/" + event.getContext() + "/");
```

Comment:
Isn't it the case that we also want to support any conditions, rather than just the gradebook one?

I know that the gradebook one is the only one that edits at the moment.